src-built-in/components/fileMenu/src/stores/fileMenuStore.js

```diff
@@ -119,25 +119,12 @@ var Actions = {
 	saveWorkspace() {
 		return new Promise(function (resolve, reject) {
 			var self = this;
-			if (FSBL.Clients.WorkspaceClient.activeWorkspace.isDirty) {
-				FSBL.Clients.DialogManager.open("yesNo",
-					{
-						question: "Your workspace \"" + FSBL.Clients.WorkspaceClient.activeWorkspace.name + "\" has unsaved changes, would you like to save?"
-					}, function (err, response) {
-						if (response.choice === "affirmative") {
-							FSBL.Clients.WorkspaceClient.saveAs({
-								force: true,
-								name: FSBL.Clients.WorkspaceClient.activeWorkspace.name
-							}, function (err1, response1) {
-								resolve(response.choice);
-							});
-						} else {
-							resolve(response.choice);
-						}
-					});
-			} else {
-				resolve();
-			}
+			FSBL.Clients.WorkspaceClient.saveAs({
```

Comment:
@carterbs I think you've gone too far. We don't want to overwrite the actual saved workspace. We just want to make sure the active workspace is saved, and then restored when the sesssion restarts. I think all you need to do is add a "dirty" flag that gets saved with the active workspace.